src/mvs_eland/E0_evaluation.py

```diff
@@ -146,7 +146,10 @@ def evaluate_dict(dict_values, results_main, results_meta):
     for group in [ENERGY_CONVERSION, ENERGY_PRODUCTION, ENERGY_CONSUMPTION]:
```

Comment:
Could have been breaked out in multiple commits (meta-comment about git workflow)